Artifacts/windows-chocolatey/install-choco-package.ps1

```diff
@@ -156,12 +171,12 @@ try
     Write-Host 'Ensuring latest Chocolatey version is installed.'
     Ensure-Chocolatey
 
-    Write-Host "Preparing to install Chocolatey packages: $Packages."
+    Write-Host "Preparing to install Chocolatey packages: $Packages with checksum flags: --allow-empty-checksums: ($AllowEmptyChecksums)  --ignore-checksums: ($IgnoreChecksums)"

```

Comment:
If you want to output input parameters, I suggest you do so as part of the initialization of this artifact and do it for all parameters, not just the 2 you just added.